static/css/application.css

```diff
@@ -0,0 +1,2 @@
+header {
+  padding: 10px; }
```

Comment:
It would be better not to commit the static/generated files as it would create needless conflicts.